python-package/xgboost/sklearn.py

```diff
@@ -258,7 +268,8 @@ def fit(self, X, y, sample_weight=None, eval_set=None, eval_metric=None,
         if evals_result:
             for val in evals_result.items():
                 evals_result_key = list(val[1].keys())[0]
-                evals_result[val[0]][evals_result_key] = val[1][evals_result_key]
+                evals_result[val[0]][evals_result_key] = val[
```

Comment:
Could you try putting this in one line or something? Continuing new line from `[` looks a little weird to me.